portfolio/src/main/webapp/aboutME.html

```diff
@@ -0,0 +1,78 @@
+            community has probably had the <br> most impact on the person that 
+            he is today.
+          </p>
+          <br>
+          <br>
+          <p style = "font-size:20px;color: grey" align = "center">
+            Adetowo comes from a very small community where everybody is 
+            pretty much the same, there's <br> not much diversity in the community
+            as majority of the people are blacks and the<br> few whites in the 
+            community are usually there for tourism. Not having an education 
+            was the <br>norm, and even though he was lucky to receive an education
+            comparable to the united states <br> high school diploma, he knew he 
+            wanted more from life. Growing up, his mother’s version of <br>Christianity
+            included a heavy dose of social responsibility. This pushed him earlier 
+            into social work and <br>education, but being a bit rebellious by nature, 
+            He took his own path getting there. He <br> was influenced on a Cultural level
+            by a history he lived through as a child: getting an <br>education wasn’t 
+            seen as a necessity, but as something only rich people could have. He <br>
+            knew the world could be a harsh place and believed that he could have a 
+            role <br>in making it better through education and he worked hard to achieve that. 
+            Consequently, he was<br> able to secure a scholarship to get his bachelor’s degree 
+            in the United States and fulfill <br> his dream of getting an education.
+          </p>
+          <br>
+          <br>
+          <p style = "font-size:20px;color: grey" align = "center">
+            Even though he lived most of his life in Nigeria where blacks are the 
+            majority, <br>there was still diversity as there are different people from 
+            different cultures and his life <br> experiences opened him up to accepting and 
+            appreciating those who are culturally diverse from him. while <br>his ability to 
+            form relationships with diverse individuals has made him more appreciative of 
+            other <br>races, he understands that this does not mean that he knows what life is 
+            like as a<br> minority. He has lived his whole life comfortably as part of the majority 
+            group in Nigeria<br>. It wasn’t until he got to college that he really started interacting 
+            with people of different<br> races and cultures. Despite the odds, here he is, and here 
+            he hopes to stay for a<br> period of time, not only culturally enriched and happy to broaden 
+            his horizons but also studying<br> hard to get a college degree to create opportunities and 
+            be an inspiration to young kids<br> like him who are bold enough to want more from life. 
+          </p>
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
Please fix this -- "No newline at the end of file"

Reply:
fixed! Thanks shenqiu!